### PR TITLE
Updated doc hostname references to ip address equivalents

### PR DESCRIPTION
**Pull request by TrimBiggs**
Replace instances of calico-01 and calico-02 when referencing hostnames with 172.17.8.101 and 172.17.8.102, respectively.

### CODE REVIEW

**Comment by tomdee** [comment]
LGTM - @robbrockbank can you take a look too?

File: docs/getting-started/GCE.md [hunk 1]
```diff
@@ -63,7 +63,9 @@ gcloud compute instances create \
   --metadata-from-file user-data=<PATH_TO_CLOUD_CONFIG>/user-data-first
 ```
 
-For the second server run:
+Open your `user-data-others` file and replace the instances of `172.17.8.101` with this private IP address of the `calico-01` server you just created.
```
Comment (robbrockbank) [inline docs/getting-started/GCE.md]: with this -> with the

Also, we don't tell the user how to determine the private IP address do we?  We explicitly tell them for AWS and DigitalOcean.

Comment (TrimBiggs) [inline docs/getting-started/GCE.md]: Correct, but I'm not sure how in GCE yet.  I can update this when I test it later.

**Comment by robbrockbank** [comment]
LGTM.

**Pull request merged**